internal/rukpak/provisioner/plain/plain.go

```diff
@@ -0,0 +1,102 @@
+package plain
+
+import (
+	"context"
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"io/fs"
+	"path/filepath"
+
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chartutil"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
+
+	"github.com/operator-framework/operator-controller/internal/rukpak/util"
+	rukpakv1alpha2 "github.com/operator-framework/rukpak/api/v1alpha2"
+)
+
+const (
+	// ProvisionerID is the unique plain provisioner ID
+	ProvisionerID = "core-rukpak-io-plain"
+
+	manifestsDir = "manifests"
+)
+
+func HandleBundleDeployment(_ context.Context, fsys fs.FS, _ *rukpakv1alpha2.BundleDeployment) (*chart.Chart, chartutil.Values, error) {
```

Comment:
As mentioned in the previous comment, this is going to change as we no longer have BundleDeployment. What do you think about removing the Handler implementations from this PR and working on them in follow ups?